src/stories/ActionMenu.stories.tsx

```diff
@@ -48,7 +48,7 @@ const ErsatzOverlay = styled.div`
 export function ActionsStory(): JSX.Element {
   const [option, setOption] = useState('Select an option')
   const onAction = (itemProps: ItemProps) => {
-    setOption(itemProps.text)
+    setOption(itemProps.text || '')
```

Comment:
If no `Item`s have `text` (i.e. if each `Item` receives custom `children`), would the selected option be distinguishable, or would `option` be `""` for any selected `Item`?

Reply:
In this particular example with ActionMenu, `text` is being used to determine which option was clicked.  In real-world use, it's more likely that that the options would specify either an `onAction` or a `key` of some sort that would make them identifiable if they used `children` to provide the content.